people/yumao/page.en.md

```diff
@@ -33,7 +33,7 @@ She made wishes to meteors occasionally. She said: "It's not because I believe t
 According to Neko, Yumao loved spicy hot pot, and always enjoyed it with rice.
 
 Yumao liked to watch stars while waiting to fall asleep at night.
-After she returned to China, due to bad weather and few spare time, she couldn't watch stars as often as she did.
+After she returned to China, due to bad weather and fewer spare time, she couldn't watch stars as often as she did.
```

Comment:
"bad weather" is not a comparison, therefore "? spare time" cannot be a comparison as well.